constants.py

```diff
@@ -112,4 +113,4 @@
     'Google',
 ]
 
-VER_SV_SEPARATION_STARTED = 73
+VER_LARGE_ENTITIES_SEPARATION_STARTED = 74
```

Comment:
nit: `VERS_LARGE_ENTITIES_SEPARATION_STARTED` so we know `VERS` stands for `VERS`ion and not `VER`y?